app/models/file_set.rb

```diff
@@ -17,9 +17,16 @@ def create_derivatives(filename, opts = {})
     # pdf for text assets
     # second thumbnail for CITI
     def create_asset_derivatives(filename)
+      # byebug
       case mime_type
       when *self.class.image_mime_types
-        Hydra::Derivatives::ImageDerivatives.create(filename, outputs: image_outputs)
+        if mime_type =~ /psd/i || mime_type =~ /vnd.adobe.photoshop/i
+          MiniMagick.with_cli(:imagemagick) do
+            Hydra::Derivatives::ImageDerivatives.create(filename, outputs: image_outputs)
+          end
+        else
+          Hydra::Derivatives::ImageDerivatives.create(filename, outputs: image_outputs)
+        end
```

Comment:
why not:
``` ruby
when *image_mime_types_with_cli
  MiniMagick.with_cli(:imagemagick) do
    Hydra::Derivatives::ImageDerivatives.create(filename, outputs: image_outputs)
  end
when *self.class.image_mime_types
  Hydra::Derivatives::ImageDerivatives.create(filename, outputs: image_outputs)
[...]

def image_mime_types_with_cli
  ["psd", "vnd.adobe.photoshop"]
end
```
This makes the method less branchy. Alternatively, you could define `image_mime_types_with_cli` in config/initializers/hydra_works.rb if you need to use these mime types outside of this class.